mm2src/mm2_main/src/wasm_tests.rs

```diff
@@ -184,3 +194,48 @@ async fn trade_test_rick_and_morty() {
     )
     .await;
 }
+
+async fn enable_z_coin_light(
```

Comment:
It would be good to combine this implementation with the below to have one common implementation
https://github.com/KomodoPlatform/komodo-defi-framework/blob/3e323df320535d9c6ac4c7b1d235501e40b1da48/mm2src/mm2_main/tests/integration_tests_common/mod.rs#L86

Reply:
done